### ISSUE TITLE
Is HDMI working???

### ISSUE THREAD

**Issue by kaludush** [RafiZz/msi-gl65-9sc-hackintosh#4]
i have same specs like you! Msi gl65 9scxr but no hdmi output

**Comment by ishangupta-1510** [comment]
Nope its not if you got it working please upload the EFI
BTW is your bluetooth working

**Comment by kaludush** [comment]
yes i got managed to fix hdmi. only thing now not working is airdrop. (airdrop shows iphone but cant communicate each other)

**Comment by Ishangupta123** [comment]
Please can you share your EFI here or on gupta.ishan.1510@gmail.com 
It would be really helpfull thank you🙏🏼

**Comment by HotJupiterO** [comment]
Hello, could you please update kext for ventura if you have a possibility of course
